packages/babel-compat-data/package.json

```diff
@@ -29,7 +29,7 @@
     "compat-data"
   ],
   "devDependencies": {
-    "electron-to-chromium": "1.3.574",
+    "electron-to-chromium": "1.3.583",
     "lodash": "^4.17.19",
     "mdn-browser-compat-data": "1.0.38"
```

Comment:
Not related: I tried updating `mdn-browser-compat-data`, too. It seems that version 1.1 is deprecated because it is renamed to `@mdn/browser-compat-data`. I will open another PR to address that.